### PR TITLE
Text: typos

### PR DESCRIPTION
**Pull request by f3l1x**
This PR fix some text typos that have been found by spellcheck #608.

### CODE REVIEW

File: en/smartobject.texy [hunk 1]
```diff
@@ -32,11 +32,11 @@ class Circle
 }
 
 $circle = new Circle;
-$circle->raduis = 10;
+$circle->radius = 10;
```
Comment (dg) [inline en/smartobject.texy]: To je záměr! :)

File: en/smartobject.texy [hunk 2]
```diff
@@ -32,11 +32,11 @@ class Circle
...
 echo $circle->getArea(); // 10² * π ≈ 314
 \--
 
-On the first look it seems that code will print out 314; but it returns 0. How is this even possible? Accidentaly, `$circle->radius` was mistyped to `raduis`. Just a small typo, which will give you a hard time correcting it, because PHP does not say a thing when something is wrong. Not even a Warning or Notice error message. Because PHP does not think it is an error.
+On the first look it seems that code will print out 314; but it returns 0. How is this even possible? Accidentaly, `$circle->radius` was mistyped to `radius`. Just a small typo, which will give you a hard time correcting it, because PHP does not say a thing when something is wrong. Not even a Warning or Notice error message. Because PHP does not think it is an error.
```
Comment (dg) [inline en/smartobject.texy]: Tohle taky :)

**Comment by dg** [comment]
Nice catches!

File: en/smartobject.texy (hunk 1 above)
Comment (f3l1x) [inline en/smartobject.texy]: Ou shit ;-) Ja vul. :-D

**Comment by f3l1x** [comment]
SmartObject vynechan ;-)

**Comment by dg** [comment]
Paráda

**Pull request merged**